content/influxdb/v2.0/influxdb-templates/monitor.md

```diff
@@ -0,0 +1,123 @@
+      ...
+      ```
+3. Verify you have access to the `/metrics` endpoint for each monitored InfluxDB OSS instance so [Telegraf](/telegraf/latest/) can collect metrics.
+
+4. Add your InfluxDB Cloud account information (url and organization) to your Telegraf configuration by doing the following:
+   1. [In your InfluxDB Cloud account](https://cloud2.influxdata.com/), go to **Load Data > Telegraf**, and click the **InfluxDB Output Plugin** button.
+   2. Copy the URL, token, organization, and bucket, close the window, and then click the **Scrape InfluxDB OSS Metrics** link.
+   3. Under `outputs.influxdb_v2`, replace `URL`, `token`, `organization`, and `bucket`, with your InfluxDB Cloud account information. These parameters use environment variables except for `bucket`.
+
+      {{% note %}}
+ Use the provided parameter value, `oss_metrics`, for the bucket to maintain compatibility with the included dashboard in the template. We recommend setting an environment variable for the API token on each machine you plan to gather metrics from using Telegraf. This token allows Telegraf to fetch the configuration and write data to your instance.
+      {{% /note %}}
+
+   4. Under `[[inputs.prometheus]]` update parameters to tell Telegraf how to scrape metrics from the `/metrics` endpoint of each InfluxDB OSS instance. You'll find the default InfluxDB OSS URL here, but if you're using unique URLs or have security set up for your `/metrics` endpoint, configure those options here and save the updated configuration.
+
+      For more information about customizing Telegraf, see [Telegraf configuration documentation](/telegraf/latest/administration/configuration/#global-tags).
+
+## Set up Telegraf
+
+Set up Telegraf to scrape metrics from InfluxDB OSS to send to your InfluxDB Cloud account.
+
+On each InfluxDB OSS instance you want to monitor, do the following:
+
+1. [In your InfluxDB Cloud account](https://cloud2.influxdata.com/), go to **Load Data > Telegraf**.
+2. Under **Scrape InfluxDB OSS Metrics**, click **Setup Instructions**, and complete the Telegraf Setup instructions.
+      {{% note %}}
+For your API token, generate a new token or use an existing All Access token. If you run Telegraf as a service, edit your init script to set the environment variable and ensure its available to the service.
+      {{% /note %}}
+   Telegraf runs quietly in the background (no immediate output appears), and Telegraf begins pushing metrics to your InfluxDB Cloud account.
+
+## View the Monitoring dashboard
+
+To see your data in real time, view the Monitoring dashboard.
+
+1. In your InfluxDB Cloud account, select **Boards** (**Dashboards**).
+
+    {{< nav-icon "dashboards" >}}
+
+2. Click **InfluxDB OSS Metrics**. Metrics appear in your dashboard.
+
+Customize your monitoring dashboard as needed. For example, send an alert in the following cases:
+- Users create a new task or bucket
+- You're testing machine limits
```

Comment:
examples for how to do?

Reply:
These aren't really dashboard related. These are task related.